src/components/SendFlow/SignButton.tsx

```diff
@@ -55,21 +55,31 @@ export const SignButton: React.FC<{
     });
 
   const onLedgerSign = async () =>
-    handleAsyncAction(async () => {
-      toast({
-        description: "Please open the Tezos app on your Ledger and approve the operation",
-        status: "info",
-        duration: 60000,
-        isClosable: true,
-      });
-      return onSubmit(
-        await makeToolkit({
-          type: "ledger",
-          account: signer as LedgerAccount,
-          network,
-        })
-      );
-    });
+    handleAsyncAction(
+      async () => {
+        toast({
+          id: "ledger-sign-toast",
+          description: "Please approve the operation on your Ledger",
+          status: "info",
+          duration: 60000,
+          isClosable: true,
+        });
+        return onSubmit(
+          await makeToolkit({
+            type: "ledger",
+            account: signer as LedgerAccount,
+            network,
+          })
+        );
+      },
+      (error: any) =>
+        ({
+          title: error.message,
+          description:
```

Comment:
IIRC there might be some styling issues when we use both title and description. It's better to just concatenate the error.message with the description

Reply:
It's quite a lot of text if we keep everything in the description. Just wanted to double-check if it's OK

<img width="920" alt="Screenshot 2024-02-23 at 13 01 16" src="https://github.com/trilitech/umami-v2/assets/150050388/4b5320ac-12bb-4a77-9cf6-6f0043f76094">

<img width="857" alt="Screenshot 2024-02-23 at 13 02 40" src="https://github.com/trilitech/umami-v2/assets/150050388/e78c7815-e1ac-4f50-abc5-ee3c80c0434e">

Reply:
Yeah, it's fine